pubspec.yaml

```diff
@@ -18,5 +18,4 @@ flutter:
     pluginClass: StoreRedirectPlugin
 
 environment:
-  sdk: ">=2.0.0-dev.28.0 <3.0.0"
-  flutter: ">=0.1.4 <2.0.0"    
```

Comment:
is it necessary to remove the flutter version?

Reply:
I don't think is useful to constrain Flutter version.
I guess the migration has been done using "dart migrate" command. It automatically removes the Flutter version.

Reply:
TBH, I've not seen the flutter constraint in any other flutter lib, so I thought this was some legacy type of thing, so I removed it. I also agree with @MisterWeeMan.

Reply:
Alright, it makes sense, thanks.